diesel/src/expression/functions/helper_types.rs

```diff
@@ -0,0 +1,24 @@
+#![allow(non_camel_case_types)]
```

Comment:
What is your argument for making these types lowercase? (Maybe I missed it)

Reply:
It's what's generated by `sql_function!`